app/orm.js

```diff
@@ -7,8 +7,6 @@ exports.DB_URL =
   `postgres://${config.common.database.username}:${config.common.database.password}
   @${config.common.database.host}:${config.common.database.port}/${config.common.database.name}`; // eslint-disable-line max-len
 
-console.log(exports.DB_URL);
```

Comment:
Oh, i made you to remove this in another PR, feel free to remote it in whatever PR you want :)